rocketpy/Motor.py

```diff
@@ -0,0 +1,1937 @@
+        d = d * (self.grainInitialHeight + self.grainSeparation)
+
+        # Calculate inertia for all grains
+        self.inertiaI = grainNumber * grainInertiaI + grainMass * np.sum(d**2)
+        self.inertiaI.setOutputs("Propellant Inertia I (kg*m2)")
+
+        # Inertia I Dot
+        # Calculate each grain's inertia I dot
+        grainInertiaIDot = (
+            grainMassDot
+            * (
+                (1 / 4) * (self.grainOuterRadius**2 + self.grainInnerRadius**2)
+                + (1 / 12) * self.grainHeight**2
+            )
+            + grainMass
+            * ((1 / 2) * self.grainInnerRadius - (1 / 3) * self.grainHeight)
+            * self.burnRate
+        )
+
+        # Calculate inertia I dot for all grains
+        self.inertiaIDot = grainNumber * grainInertiaIDot + grainMassDot * np.sum(
+            d**2
+        )
+        self.inertiaIDot.setOutputs("Propellant Inertia I Dot (kg*m2/s)")
+
+        # Inertia Z
+        self.inertiaZ = (
+            (1 / 2.0)
+            * self.mass
+            * (self.grainOuterRadius**2 + self.grainInnerRadius**2)
+        )
+        self.inertiaZ.setOutputs("Propellant Inertia Z (kg*m2)")
+
+        # Inertia Z Dot
+        self.inertiaZDot = (1 / 2.0) * self.massDot * (
+            self.grainOuterRadius**2 + self.grainInnerRadius**2
+        ) + self.mass * self.grainInnerRadius * self.burnRate
+        self.inertiaZDot.setOutputs("Propellant Inertia Z Dot (kg*m2/s)")
+
+        return [self.inertiaI, self.inertiaZ]
+
+    def exportEng(self, fileName, motorName):
+        """Exports thrust curve data points and motor description to
+        .eng file format. A description of the format can be found
+        here: http://www.thrustcurve.org/raspformat.shtml
+
+        Parameters
+        ----------
+        fileName : string
+            Name of the .eng file to be exported. E.g. 'test.eng'
+        motorName : string
+            Name given to motor. Will appear in the description of the
+            .eng file. E.g. 'Mandioca'
+
+        Returns
+        -------
+        None
+        """
+        # Open file
+        file = open(fileName, "w")
+
+        # Write first line
+        file.write(
+            motorName
+            + " {:3.1f} {:3.1f} 0 {:2.3} {:2.3} RocketPy\n".format(
+                2000 * self.grainOuterRadius,
+                1000
+                * self.grainNumber
+                * (self.grainInitialHeight + self.grainSeparation),
+                self.propellantInitialMass,
+                self.propellantInitialMass,
+            )
+        )
+
+        # Write thrust curve data points
+        for time, thrust in self.thrust.source[1:-1, :]:
+            # time, thrust = item
+            file.write("{:.4f} {:.3f}\n".format(time, thrust))
+
+        # Write last line
+        file.write("{:.4f} {:.3f}\n".format(self.thrust.source[-1, 0], 0))
+
+        # Close file
+        file.close()
+
+        return None
+
+    def info(self):
```

Comment:
Also wondering why this method was overwritten...

Reply:
Kept this one only on the base class as well. 

Regarding `allInfo`, on the other hand, it seemed better to be kept class-dependent, since there will be lots of motor-type specific info displayed by the method.